### DISCUSSION TITLE
Hide a resource variable from the documentation

### DISCUSSION THREAD

**Discussion by prullenbak** [dedoc/scramble#522] [Q&A]
Hello, 

Nice package! But I have a problem: I have some variables in my resources that are only visible by admins. These are hidden in my resources with (simplified)
`'field' => $this->when(auth()->user()->can('some-permission'), $this->field),`
This hides the field from my normal users fine, however, the 'field' does show up in the scramble documentation. I'd like to be able to hide this field from the docs. Is this possible? I tried the @ignoreParam annotation but that didn't work.

Simply removing the field from the resource won't work, because I'm using the resources in my application to send data to my Inertia front-end. Of course, making a separate resource for this is an option, but If I could simply hide the variable with an annotation, that would be way simpler :)

Thanks!

**Comment by vismutx** [comment]
I found
@ignoreParam
should be used in every place where you use it

So I've added @ignoreParam into rules and into code where I get this param in code
